Jenkinsfile.meta

```diff
@@ -40,6 +40,17 @@ node {
 			sh '''
 				git submodule update --remote --merge .doi
 				git submodule update --remote --merge .scripts
+
+				# TODO once "repos_anti_subset" in "doi.jq" is empty, we can remove this (and all associated usages of "subset.txt" can just be "--all" or go away completely)
+				# in all the places we need to interact with our "subset" it's a lot easier to have an explicit list of what's included, so we'll continue to generate "subset.txt" until it contains the full set
+				bashbrew list --all --repos | jq -L.scripts -rsR '
+					include "doi";
+					rtrimstr("\n")
+					| split("\n")
+					| . - repos_anti_subset
+					| join("\n")
+				' > subset.txt
+				git add subset.txt
```

Comment:
So, now that this will generate `subset.txt` from the data in `doi.jq`, maybe we include some or all of the repos in https://github.com/docker-library/meta/pull/31, https://github.com/docker-library/meta/pull/34, https://github.com/docker-library/meta/pull/35, https://github.com/docker-library/meta/pull/36 as part of the changes here (and add a "closes")? That way we can exercise this bit in the first run after merge and also not put those on hold or forget if we decide to merge any of them before this is merged.

Reply:
Updated :+1: